src/main.py

```diff
@@ -65,17 +65,17 @@ def setup_escambo_actions(self, win):
             ["<primary>r"],
         )
 
-    def on_about_action(self, widget, _):
+    def on_about_action(self, *args):
```

Comment:
Thanks for this contribution.
Please, don't do things the PR doesn't say it is.